### ISSUE TITLE
Server performance

### ISSUE THREAD

**Issue by jwhite2000** [mozilla/speech-proxy#89]
Is there any way to speed up the server's response time. I am calling the api here https://speaktome-2.services.mozilla.com/ using the speaktome javascript modules developed by Mozilla. It is taking between 3 and 6 seconds to get a response.

**Comment by milescrabill** [comment]
Yeah, that's pretty abysmal. There are a few issues with the current server side architecture that we're looking at addressing. We're also moving towards a new speech solution based on [Deepspeech](https://github.com/mozilla/deepspeech) that will run on GPUs instead of CPUs. The hope is that that will speed things up significantly.

**Comment by flozi00** [comment]
how can we speed up the transition to deepspeech?
Any possibilitys to support ?

**Comment by flozi00** [comment]
> Yeah, that's pretty abysmal. There are a few issues with the current server side architecture that we're looking at addressing. We're also moving towards a new speech solution based on [Deepspeech](https://github.com/mozilla/deepspeech) that will run on GPUs instead of CPUs. The hope is that that will speed things up significantly.


What about clustering like solution ?

Client asks --> server returns IP --> client use ip instead of https://speaktome-2.services.mozilla.com/ as static server

Then you have a load balancing option, with only one more Handshake
